conf/messages.en

```diff
@@ -32,7 +32,7 @@ site.continueToPspDashboard = Continue to your practitioner overview
 error.title.prefix = Error:
 error.summary.title = There is a problem
 
-footer.links.support_links = Cysylltiadau cymorth
+footer.links.support_links = Help contacts
```

Comment:
Do we have a separate Welsh language version for the same link? (Do we need one...?)

Reply:
As of now, we don't need it. but once we start supporting it we need to create a separate message file supporting welsh.

Reply:
Thanks @Ashwani1403